.travis.yml

```diff
@@ -9,6 +9,10 @@ before_install:
   - pip install pytest-cov
 install:
   - pip install -r requirements.txt
+before_script: # configure a headless display to test plot generation
+  - "export DISPLAY=:99.0"
+  - "sh -e /etc/init.d/xvfb start"
+  - sleep 3 # give xvfb some time to start
```

Comment:
I think we'll want to get rid of this eventually.  Pyke tools should be able to run on machines without displays!